packages/material-ui-lab/src/themeAugmentation/index.js

```diff
@@ -0,0 +1 @@
+export default {};
```

Comment:
Seems like such a synthetic file is a bad solution. We do not actually need that.